drivers/usb/common/nrf_usbd_common/nrf_usbd_common.c

```diff
@@ -834,9 +834,10 @@ static void usbd_dmareq_process(void)
 {
 	if ((m_ep_dma_waiting & m_ep_ready) &&
 	    (k_sem_take(&dma_available, K_NO_WAIT) == 0)) {
+		const bool active = !nrf_usbd_common_suspend_check();
```

Comment:
Would not `const bool suspended = nrf_usbd_common_suspend_check();`  be clearer?

Reply:
I wanted the condition inside `while` to be clear. If `suspended` is used instead of `active` then the condition would have to be changed to `while (!suspended && 0 != (req = m_ep_dma_waiting & m_ep_ready))` which I think is less clear.

Reply:
Changed to `low_power` to be more clear. It is not the USB Suspend that prohibits DMA transfers, but rather Low Power mode that the driver enables while suspended. Driver may choose to not enable the Low Power mode, but then the suspend current is higher.

Reply:
okay, thanks